packages/ketcher-react/src/script/editor/tool/enhanced-stereo.ts

```diff
@@ -46,6 +46,7 @@ class EnhancedStereoTool {
     this.changeAtomsStereoAction().then(
       (action) => action && editor.update(action)
     )
+    this.isNotActiveTool = true
```

Comment:
That works, but maybe let's make it even more obvious? It should never be an active tool, so let's make it always true:

```typescript
class EnhancedStereoTool {
  editor: Editor
  stereoAtoms: Array<number>
  isNotActiveTool: true

  constructor(editor) {
    const selection = editor.selection()
    this.editor = editor
    this.isNotActiveTool = true
...
```

And we can remove all the other lines with `this.isNotActiveTool = true` after that.